vue-module-6/src/main.js

```diff
@@ -1,5 +1,33 @@
 import { createApp } from 'vue'
 import App from './App.vue'
 import router from './router'
+import { createI18n } from 'vue-i18n'
 
-createApp(App).use(router).mount('#app')
+const i18n = createI18n({
```

Comment:
Can we do this in a separate file instead of doing here because we might use our app for multiple regions? take reference from our apps for learning otherwise it is ok for now.

Reply:
Looking forward to correcting it!